build/index/index.js

```diff
@@ -1,4 +1,64 @@
-function reserveFlight(element) {
-    element.innerHTML = "Reserved";
-    element.parentElement.parentElement.classList.add("reserved");
-}
\ No newline at end of file
+document.onload(FlightCard());
+function FlightCard() {
+  const data = [
+    {
+      Id: 0,
+      direction: 'Warsaw (WAW) - Dubaj (DXB)',
+      flight: '29-10-2023',
+      return: '30-10-2023',
+      price: 23,
+      image: 'build/index/flghtsImg/dubai.jpg',
+    },
+    {
+      Id: 1,
+      direction: 'Warsaw (WAW) - Miami (MIA)',
+      flight: '5-11-2023',
+      return: '11-11-2023',
+      price: 323,
+      image: 'build/index/flghtsImg/miami.jpg',
+    },
+    {
+      Id: 2,
+      direction: 'Warsaw (WAW) - Mubai (BOM)',
+      flight: '14-10-2023',
+      return: '1-11-2023',
+      price: 113,
+      image: 'build/index/flghtsImg/mubai.jpg',
```

Comment:
teraz strona główna generuje się sama na podstawie danych jakie otrzyma

Reply:
ale za to jakbyśmy znaleźli jakieś api które posiada dane o lotach to byśmy mogli wygenerować cała strone lotów z danych które byśmy dostali zamiast hard codedować 3